db/migrate/20170915032724_add_outquotes_to_article.rb

```diff
@@ -1,5 +0,0 @@
-class AddOutquotesToArticle < ActiveRecord::Migration[5.1]
```

Comment:
Careful. Don't get into the habit of deleting migrations. Instead, create a new migration that undoes the other's changes. If you delete migrations, then you have an issue when you try to migrate existing databases to the new schema